### DISCUSSION TITLE
Customize web page

### DISCUSSION THREAD

**Discussion by ghost** [searxng/searxng#1436] [General]
Hi, 
Where are the HTML files to be able to customize the search engine?
For example on the "home page" i would like to change the searxng to something else.

**Comment by mrpaulblack** [comment]
Hi @asidscript ,

The CSS and JS files for styling (so this includes the images and the SearXNG logo on the homepage) are located here: https://github.com/searxng/searxng/tree/master/searx/static/themes/simple/



The HTML files are located here (there is some non HTML stuff in there, that is because we use a template engine called jinja2; basically the server builds those HTML files when a user requests them, while its running): https://github.com/searxng/searxng/tree/master/searx/templates/simple

**Comment by return42** [comment]
If you want to modify CSS please read https://docs.searxng.org/dev/quickstart.html

If you are not familiar with git, jinja2, less and development in general, I can't recommend to customize.

**Comment by ghost** [comment]
Markus, Paul
Thank you guys. this is awesome.

Regards
Dan

**Comment by jebofponderworthy** [comment]


**Reply by return42** [reply]
Very danger .. don't do it .. you are not able to switch from POST to GET and you have trouble with other settings .. it will brake with next update that implements a patch of the UI!

**Reply by return42** [reply]
@jebofponderworthy you don't seem to understand what you are doing.  I will not tolerate such a dangerous solution here in the forum, as it could tempt other inexperienced users to simply copy it. I will now delete your suggestion to avoid this.  My recommendation for the future would be not to manipulate software without understanding what you are doing.
